### DISCUSSION TITLE
Add mode/custom display aspect ratio 16:9, 21:9 32:9 48:9 (triplets)

### DISCUSSION THREAD

**Discussion by carlosplazaaguilera** [djyt/cannonball#131] [Ideas]
It will be great could to play in a simracing simulator with triplet monitor.
Thanks.

**Comment by MichaelRPowell** [comment]
Personally, I understand supporting higher resolutions. This could allow the game to run on a [video wall](https://www.userful.com/hubfs/Imported_Blog_Media/72dpi-hallwayvideowall-large.jpg). That seems like a niche use, but would be neat and I assume not difficult to add.

I'm personally not sure that three monitors on a _racing simulator_ would be great or even easy to implement for OutRun. [This is what I am picturing when you say simulator.](https://www.simxperience.com/web/image/product.template/18/image_1024?unique=be1e4c9)

The issue is that the game would have to account for the angle of the side monitors (a video wall wouldn't require that). I am curious, and unsure, if the sprite scaling to simulate driving in 3D would work well with angled side monitors. What I mean is the sprites are only viewed from straight on. If you display those sprites on the side monitors they would have to be rotated to face the user to keep their appearance. Otherwise, you would see they're flat sprites, and it would give the impression you are driving by cardboard cutouts of everything.

If you rotate sprites on the side monitors this creates the visual issue of everything you pass rotating to face towards you on either side. Trees wouldn't stand out (they look roughly the same from all angles), but buildings and arches I assume would look strange. I only mention this because I could not find any games using sprite scaling with three angled monitors. Three monitor racing games seem to be 3D. I am not sure how difficult that would be to overcome to make OutRun look good on a simulator.

I am curious if the author can come up with a solution, but felt I should point out the technical hurdles that might prevent/delay that kind of implementation. :)

**Reply by carlosplazaaguilera** [reply]
Totally agree. It is a 2D game so it only works correctly with an orthogonal proyections.  I don´t want to make it 3D or VR support It will result in a fully remake, resulting like Doom with his rotanting sprites....
 
So what I propose is only to add customs resolutions options, prefered with custom aspect ratio. With that options (resolutions ans aspect ratio) it will work for a video wall with 48:9 (16:9 x3) aspect ratio, 21:9, 1440p, 4K.... and legancy and future proof resolutions or aspect ratios. Only fully compatibility with any actual or future display systems.

In fact in Dirt Rally, F1 2020, City Car driving School Simulator and other simulation products doesn´t support 3 angled monitors, only expand the FOV horizontally to fill the three monitors/displays. This is not correct or aceptale for 3d simulator, but we are talking about a 2D game, three perspectives are not necessary, only an expansion of the gaming area.
